docs/installation.md

```diff
@@ -199,36 +199,35 @@ object MyScalafmtPlugin extends AutoPlugin {
 include ".scalafmt-common.conf"
 ```
 
-## CLI
+## CLI / Homebrew
```

Comment:
This breaks existing links to the `#cli` anchor. I think it's better to keep the `Homebrew` header unchanged

Reply:
changed. (i had checked whether `#cli` was used within the file but you're right, it might be referred to externally.)